M6-Backend/MockFacility.test.js

```diff
@@ -0,0 +1,210 @@
+//Test Set 1 --> Testing the interface specific for module facility
+const interfacespecific = require('./interfacespecific');
```

Comment:
Codacy has a fix for the issue: [Replace `'./interfacespecific'` with `"./interfacespecific"`](https://app.codacy.com/gh/HizanLi/CPEN321/pullRequest?prid=9816566)
```suggestion
const interfacespecific = require("./interfacespecific");
```